modules/installation-azure-stack-hub-config-yaml.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc
+
+[id="installation-azure-stack-hub-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for Azure Stack Hub
+
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. Use it as a resource to enter parameter values into the installation configuration file that you created manually.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+credentialsMode: Manual <1>
+controlPlane: <2>
+  hyperthreading: Enabled <3>
+  name: master
+  platform:
+    azure:
+      osDisk:
+        diskSizeGB: 1024 <4>
+      type: Standard_D8s_v3
+  replicas: 3
+compute: <2>
+- hyperthreading: Enabled <3>
+  name: worker
+  platform:
+    azure:
+      type: Standard_D2s_v3
+      osDisk:
+        diskSizeGB: 512 <4>
```

Comment:
Not sure it makes sense to include `hyperthreading` or `platform` here or for the masters, since these aren't managed by machinesets, right? Do we do this for other upi platforms?

Reply:
`hyperthreading` was included in some UPI install docs, like [vSphere](https://docs.openshift.com/container-platform/4.8/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-config-yaml_installing-vsphere). I'll remove these here, though, if they're unnecessary.